src/library/Uno.Material/Uno.Material.csproj

```diff
@@ -28,12 +28,8 @@
 		<Compile Update="**\*.xaml.cs">
 			<DependentUpon>%(Filename)</DependentUpon>
 		</Compile>
-		<Content Include="Assets\MaterialDeterminate.json">
-		  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Was removing the `CopyToOutput` change intentional?

Reply:
yeah, I don't think it was needed